ember/package.json

```diff
@@ -12,75 +12,103 @@
+    "ember-can": "^3.1.0",
+    "ember-changeset": "^v4.0.0-beta.5",
+    "ember-changeset-validations": "^v4.0.0-beta.3",
+    "ember-cli": "~3.28.5",
+    "ember-cli-app-version": "^5.0.0",
+    "ember-cli-babel": "^7.26.10",
     "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-htmlbars": "^5.7.1",
-    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-htmlbars": "^5.7.2",
+    "ember-cli-inject-live-reload": "^2.1.0",
+    "ember-cli-mirage": "^2.2.0",
     "ember-cli-moment-shim": "^3.8.0",
     "ember-cli-sass": "^10.0.1",
-    "ember-cli-showdown": "^4.5.0",
+    "ember-cli-showdown": "^6.0.0",
     "ember-cli-sri": "^2.1.1",
+    "ember-cli-string-helpers": "^6.1.0",
     "ember-cli-uglify": "^3.0.0",
-    "ember-composable-helpers": "^4.4.1",
-    "ember-concurrency": "^1.3.0",
-    "ember-concurrency-decorators": "^2.0.3",
-    "ember-data": "~3.27.0",
-    "ember-emeis": "~1.0.0",
-    "ember-engines": "^0.8.6",
+    "ember-cli-update": "^1.0.1",
+    "ember-composable-helpers": "^4.5.0",
+    "ember-concurrency": "^2.2.1",
+    "ember-data": "~3.28.6",
+    "ember-emeis": "^5.0.0",
+    "ember-engines": "^0.8.20",
     "ember-export-application-global": "^2.0.1",
-    "ember-fetch": "^6.3.1",
-    "ember-in-viewport": "^3.9.0",
-    "ember-intl": "^5.5.1",
+    "ember-fetch": "^8.1.1",
+    "ember-in-viewport": "^3.10.3",
+    "ember-intl": "^5.7.2",
     "ember-load-initializers": "^2.1.2",
-    "ember-math-helpers": "^2.16.0",
+    "ember-math-helpers": "^2.17.3",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-moment": "^8.0.1",
+    "ember-moment": "^10.0.0",
     "ember-parachute": "^1.0.2",
     "ember-pikaday": "^3.0.0",
-    "ember-power-select": "^4.1.5",
+    "ember-power-select": "^5.0.4",
     "ember-promise-helpers": "^1.0.9",
-    "ember-qunit": "^4.6.0",
-    "ember-resolver": "^8.0.0",
-    "ember-source": "~3.27.1",
-    "ember-template-lint": "^2.21.0",
-    "ember-uikit": "^2.1.2",
+    "ember-qunit": "^5.1.5",
+    "ember-resolver": "^8.0.3",
+    "ember-source": "~3.28.8",
+    "ember-template-lint": "^3.15.0",
+    "ember-truth-helpers": "^3.0.0",
+    "ember-uikit": "^5.0.0",
+    "ember-validated-form": "^5.3.0",
     "eslint": "^7.32.0",
     "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-ember": "^10.5.1",
-    "eslint-plugin-import": "^2.24.0",
+    "eslint-plugin-ember": "^10.5.8",
+    "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.4.0",
-    "eslint-plugin-qunit": "^6.2.0",
-    "graphql": "^14.0.2",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-qunit": "^7.2.0",
+    "faker": "^6.6.6",
+    "file-saver": "^2.0.5",
+    "graphql": "^15.7.1",
     "loader.js": "^4.7.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.3.2",
+    "prettier": "^2.5.1",
+    "qunit": "^2.17.2",
     "qunit-dom": "^1.6.0",
-    "sass": "^1.32.13"
+    "sass": "^1.43.4",
+    "webpack": "^5.60.0"
```

Comment:
Pin all dependencies to an exact version

Reply:
Done